easybuild/easyconfigs/s/snptest/snptest-2.5.6.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'Tarball'
+
+name = 'snptest'
+version = '2.5.6'
+
+homepage = 'https://www.chg.ox.ac.uk/~gav/snptest/'
+description = """SNPTEST is a program for the analysis of single SNP association in genome-wide studies.
+ The tests implemented include
+
+ Binary (case-control) phenotypes, single and multiple quantitative phenotypes
+ Bayesian and Frequentist tests
+ Ability to condition upon an arbitrary set of covariates and/or SNPs.
+ Various different methods for the dealing with imputed SNPs.
+"""
+
+toolchain = SYSTEM
+
+source_urls = ['https://www.well.ox.ac.uk/~gav/resources/']
+sources = ['snptest_v2.5.6_CentOS_Linux7.8-x86_64_dynamic.tgz']
+checksums = ['c2c829def961dd2f6377c388d8aa22cab17945961c47e39c4a94493466c0a52e']
+
```

Comment:
Should we add a postinstall command to symlink `snptest` to `snptest_v2.5.6`?

```
postinstallcmds = ["cd %(installdir)s && ln -s snptest_v2.5.6 snptest"]
```

Reply:
Yes, would be a friendlier approach.